src/ucp/core/ucp_ep.c

```diff
@@ -642,7 +642,8 @@ ucs_status_t ucp_worker_mem_type_eps_create(ucp_worker_h worker)
         status = ucp_ep_create_to_worker_addr(worker, &ucp_tl_bitmap_max,
                                               &local_address,
                                               UCP_EP_INIT_FLAG_MEM_TYPE |
-                                              UCP_EP_INIT_FLAG_INTERNAL,
+                                              UCP_EP_INIT_FLAG_INTERNAL |
+                                              UCP_EP_INIT_ALLOW_KA_AUX_TL,
```

Comment:
looks like these are changes from another PR

Reply:
Yeah, I messed up the commit, need to figure out how to back it out

Reply:
i think i have the commit problem fixed now.